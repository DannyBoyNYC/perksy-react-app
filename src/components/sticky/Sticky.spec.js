const React = require('react');

const Sticky = require('./index').default;

describe('<Sticky.* />', () => {
  let component;
  const requiredProps = {
    children: <div className="child" />
  };

  describe('<Sticky.Bottom />', () => {
    beforeEach(() => {
      component = shallowUntilTarget(
        <Sticky.Bottom {...requiredProps} />,
        'Sticky'
      );
    });

    it('should render correctly with all required props', () => {
      expect(component).toMatchSnapshot();
    });
  });

  describe('<Sticky.Top />', () => {
    beforeEach(() => {
      component = shallowUntilTarget(
        <Sticky.Top {...requiredProps} />,
        'Sticky'
      );
    });

    it('should render correctly with all required props', () => {
      expect(component).toMatchSnapshot();
    });
  });
});
