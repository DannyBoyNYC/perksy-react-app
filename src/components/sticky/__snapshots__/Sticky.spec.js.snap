// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sticky.* /> <Sticky.Bottom /> should render correctly with all required props 1`] = `
<div
  className="Sticky-root-0-1-1 Sticky-bottom-0-1-2"
>
  <div
    className="child"
  />
</div>
`;

exports[`<Sticky.* /> <Sticky.Top /> should render correctly with all required props 1`] = `
<div
  className="Sticky-root-0-1-1 Sticky-top-0-1-3"
>
  <div
    className="child"
  />
</div>
`;
